src/fundus/publishers/de/__init__.py

```diff
@@ -258,6 +259,16 @@ class DE(PublisherEnum):
         parser=TazParser,
     )
 
+    Heise = PublisherSpec(
+        name="Heise",
+        domain="https://www.heise.de",
+        sources=[
+            RSSFeed("https://www.heise.de/rss/heise.rdf"),
```

Comment:
There is a sitemap referenced at the robots.txt. Is there a reason you didn't add it?

Reply:
Thanks for adding 👍